File: static-analyzer/test-snippets/LogDivide.java
public class LogDivide {
    public static int countHalvings(int n) {
        int steps = 0;
        while (n > 1) {
            n = n / 2;
            steps++;
        }
        return steps;
    }

    public static void main(String[] args) {
        System.out.println(countHalvings(1024));
    }
}

File: static-analyzer/test-snippets-csharp/LogDivide.cs
public class LogDivide
{
    public static int CountHalvings(int n)
    {
        int steps = 0;
        while (n > 1)
        {
            n = n / 2;
            steps++;
        }
        return steps;
    }

    public static void Main()
    {
        System.Console.WriteLine(CountHalvings(1024));
    }
}
